test/marked-element.html

```diff
@@ -249,13 +249,11 @@
         markedElement.callback = function(err, text) {
           assert.equal(text, '<p>Some content</p>\n');
           return '<div>Overridden!</div>';
-        }
-        expect(outputElement.innerHTML).to.equal(proofElement.innerHTML);
+        } expect(outputElement.innerHTML).to.equal(proofElement.innerHTML);
```

Comment:
This is weird

Reply:
Ah it needs a semicolon after the curly brace. Good catch